### DISCUSSION TITLE
Exporter not working since install on Monday

### DISCUSSION THREAD

**Discussion by dubefab** [akvorado/akvorado#537] [Q&A]
Hi!
Since installing akvorado, I have increasing errors in those fields, and it isn’t stopping. On the web UI, I do not see my exporter (the device I configured for netflow).

When I run:
```

user@akvorado:/opt/akvorado/config$ curl -s http://127.0.0.1:8080/api/v0/inlet/metrics | grep '^akvorado_inlet' | grep _errors
akvorado_inlet_core_flows_errors{error="SNMP cache miss",exporter="192.168.100.1"} 924721
akvorado_inlet_core_flows_errors{error="sampling rate missing",exporter="192.168.100.1"} 924721
akvorado_inlet_flow_decoder_netflow_errors_count{error="template not found",exporter="192.168.100.1"} 58
```

Only the "template not found” error isn’t increasing anymore.

When I run:
`curl -s http://127.0.0.1:8080/api/v0/inlet/flows\?limit=1`

Nothing happens.

When I run
```
user@akvorado:/opt/akvorado/config$ curl -s http://127.0.0.1:8080/api/v0/inlet/metrics | grep '^akvorado_inlet_flow_input_udp_packets'
akvorado_inlet_flow_input_udp_packets{exporter="192.168.100.1",listener="0.0.0.0:2055",worker="4"} 71139

```
My /config/inlet.yaml about snmp is:
```
snmp:
  workers: 10
  communities:
    ::/0: intel-infra
```

The rest is unchanged from default.

Config on a ASA 5512-X Cisco firewall (192.168.100.1):

```
flow-export destination INSIDE 10.100.141.81 2055
flow-export template timeout-rate 15
flow-export delay flow-create 60
```

```
class-map flow_export_class
 match any

policy-map global_policy
  class flow_export_class
  flow-export event-type all destination 10.100.141.81
```    <— akvorado ip

<img width="1410" alt="SC 2023-03-01 at 11 21 25@2x" src="https://user-images.githubusercontent.com/36094769/222228750-d535781b-6bf2-45f2-ada4-0a7a1c81380c.png">

<img width="1685" alt="SC 2023-03-01 at 11 21 34@2x" src="https://user-images.githubusercontent.com/36094769/222228781-5208048f-dbee-4185-930d-b2acd1461150.png">

<img width="878" alt="image" src="https://user-images.githubusercontent.com/36094769/222228985-98f763c9-9517-433a-a471-b379dcd14cf7.png">

Thanks for the help!!!

**Comment by dubefab** [comment]
sorry I missed up the images.
here you go

<img width="1685" alt="SC 2023-03-01 at 11 21 34@2x" src="https://user-images.githubusercontent.com/36094769/222229910-0add4275-ad22-41e3-afcd-3b9594dce9b8.png">
<img width="1410" alt="SC 2023-03-01 at 11 21 25@2x" src="https://user-images.githubusercontent.com/36094769/222229928-11b9fbf0-1235-4e6b-8bf0-89386f4ea46d.png">
<img width="878" alt="image" src="https://user-images.githubusercontent.com/36094769/222229961-3d18ea76-fc79-4707-a8f0-90cf4e32e53c.png">

**Comment by vincentbernat** [comment]
Sampling rate missing, you need to specify the default sampling rate in the inlet configuration. For SNMP cache miss, it may be because you didn't allow SNMP connection to your firewall. You can grep on `poller_failure` for metrics.

**Reply by dubefab** [reply]
I added:
default-sampling-rate: 15000

Would that be an ok rate for a Cisco ASA?

**Comment by dubefab** [comment]
It looks like akvorado now sees traffic from my firewall!!!
But on the VIsualize page it shows no traffic ... what you see is from before I removed the demo data.
Can you tell me what you think is missing? I don't have snmp or template errors anymore.
<img width="1964" alt="image" src="https://user-images.githubusercontent.com/36094769/223209771-d4cc0eb3-178f-4c82-b118-a1a38ad9a01e.png">

**Reply by vincentbernat** [reply]
What happens if you remove the filter?

**Reply by dubefab** [reply]
it still shows nothing...

**Reply by vincentbernat** [reply]
On the homepage, do you see a latest flow?

**Reply by dubefab** [reply]
That's what I see on homepage:
![SC 2023-03-13 at 09 52 03@2x](https://user-images.githubusercontent.com/36094769/224755615-ff4ab75b-1c51-4866-a8d4-6cc9e8e5f8f7.png)

**Reply by vincentbernat** [reply]
Bytes and Packets is 0, which is odd.

**Reply by dubefab** [reply]
Any idea why ... ? I've pasted all the config here. You can ask me anything else.

**Reply by vincentbernat** [reply]
Most likely, your exporter does not export this info. You could check that with a network capture (with tcpdump and wireshark).
